modules/lex_calendar/templates/calendar-page.html.twig

```diff
@@ -16,25 +34,82 @@ window.addEventListener( 'load', function(){
+        center: '',
+        right: '',
       },
       editable: false,
       eventsLimit: true,
+      defaultView: 'month',
       events: '/calendar/{{ fc_callback }}',
       defaultDate: '{{ "now"|date("Y-m-d") }}',
       views: {
         month: {
-          titleFormat: 'MMMM'
+          titleFormat: 'MMMM YYYY'
         },
         day: {
           titleFormat: 'dddd, MMMM d'
         }
+      },
+      eventAfterAllRender: function() {
+        var view = $('#calendar').fullCalendar('getView');
+
+        // reset calendar dots
+        $('#calendar').find('.fc-event').remove();
+
+        // manage dots
+        var events = $('#calendar').fullCalendar('clientEvents');
+
+        for (var i = 0; i < events.length; i++) {
+            var event = events[i];
+            var name = event.title;
+            var now = event.start.clone();
+
+        // loop through event dates
+            var $dot = $('<span class="fc-day-grid-event fc-event fc-start fc-not-end dot-event " title=""></span>');
+            $dot.css('background-color', event.color);
+            // append to calendar day
+            $('.fc-day-top[data-date="'+ now.format('YYYY-MM-DD') +'"]').append($dot);
+            now.add(1, 'days');
+        }
+      }
+    });
+
+    $('#sidebar-calendar').fullCalendar({
+      header: {
+        left: 'prev,today,next',
+        center: '',
+        right: '',
+      },
+      editable: false,
+      eventsLimit: true,
+      defaultView: 'listMonth',
+      events: '/calendar/{{ fc_callback }}',
+      defaultDate: '{{ "now"|date("Y-m-d") }}',
+      listDayFormat: 'MMM D',
+      eventAfterAllRender: function() {
+        var view = $('#sidebar-calendar').fullCalendar('getView');
+                
+        $('#sidebar-calendar').find('.fc-list-item').remove();
+        $('#sidebar-calendar').find('.fc-list-heading-alt').remove();
+
+        var events = $('#sidebar-calendar').fullCalendar('clientEvents');
+
+        for (var i = 0; i < events.length; i++) {
+          var event = events[i];
+          var name = event.title;
+          var now = event.start.clone();
+
+          var $listBlock = $('<div class="list-event-container"><a href="' + event.url + '"><span class="fc-day-grid-event fc-event fc-start fc-not-end dot-event list-dot " style="background-color:' + event.color +'" title=""></span><div class="list-text-container"><span class="list-event-title">' + event.title + '</span><span class="list-event-date">' + event.start.format('MMMM D, YYYY') + '</span><span class="list-event-time">' + event.start.format('h:mm a') + '</span><span class="details-link">Details <i class="fas fa-chevron-right"></i></div></a></div>');
+
+          // append to calendar day
+          $('.fc-list-heading[data-date="'+ now.format('YYYY-MM-DD') +'"]').append($listBlock);
+          now.add(1, 'days');
+        }
       }
     });
 
-    $('#mobileCalendar').fullCalendar({
+    /*$('#mobileCalendar').fullCalendar({
```

Comment:
Are you keeping this? Kinda looks like you aren't. Did you check the calendar functionality on the phone?

Reply:
Yeah, not keeping it. It's only there at the moment to preserve just in case it's somehow needed for future iteration, but it's probably safe to just remove it. The main and sidebar calendars work on mobile now.